src/player.cpp

```diff
@@ -306,7 +306,11 @@ void Player::Update(bool update_scene) {
 	int speed_modifier = GetSpeedModifier();
 
 	for (int i = 0; i < speed_modifier; ++i) {
+		bool was_transition_pending = Graphics::IsTransitionPending();
 		Graphics::Update();
+		if (was_transition_pending && !Graphics::IsTransitionPending() && Scene::instance->IsInitialized()) {
```

Comment:
I don't find this intuitive. Could you add a comment to explain that this is used by Scene Map to handle scene calling before the first update?

Reply:
I don't find it intuitive at all. In fact I hate it, but I don't know how else to solve this.

I will add the comment.